pending-documentation.md

```diff
@@ -648,6 +648,72 @@ ambiguity during parsing, so care should be taken to ensure that named
 nonterminals are sufficiently unique from one another to prevent such
 ambiguities. Of course, the compiler will generate a warning in this case.
 
+### `concrete` attribute, `#isConcrete` and `#isVariable` function
+
+**NOTE**: `#isConcrete` and `#isVariable` are _not_ supported by the Haskell
```

Comment:
We should emphasize they _will never be_ supported by the Haskell backend.

Reply:
Done.